Cairo Desktop/Cairo Desktop/DesktopNavigationToolbar.xaml

```diff
@@ -3,53 +3,61 @@
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
         xmlns:l10n="clr-namespace:CairoDesktop.Localization;assembly=CairoDesktop.Localization"
         WindowStyle="None"
-    ResizeMode="NoResize"
-    ShowInTaskbar="False"
-        ShowActivated="False"
-            AllowsTransparency="True"
+        ResizeMode="NoResize"
+        ShowInTaskbar="False"
+        Focusable="False"
```

Comment:
This should be ShowActivated

Reply:
Odd

Reply:
Probably just from a conflict or something. I had removed focusable and replaced it with showactivated